ale/isd_generate.py

```diff
@@ -57,6 +60,13 @@ def main():
         action="store_true",
         help="Display information as program runs."
     )
+    parser.add_argument(
+        "-c", "--compress",
+        action="store_true",
+        help="Output a compressed isd json file with .br file extension. "
+             "Ale uses the brotli compression algorithm. "
+             "To decompress an isd file run: python -c \"import ale.isd_generate as isdg; isdg.decompress_json('/path/to/isd.br')\""
+    )
```

Comment:
Cool! So using this, one could write directly to compressed. Then a SET could read the compressed ISD straight to memory and use it?

Reply:
Cool! I didn't mean for this comment to spill into more work over there! Much appreciated.